test/tchannel_bridge.js

```diff
@@ -122,7 +122,6 @@ describe ('test tchannel span bridge', () => {
                     let serverSpanTags = {};
                     serverSpanTags[opentracing.Tags.PEER_SERVICE] = 'echo';
                     serverSpanTags[opentracing.Tags.SPAN_KIND] = opentracing.Tags.SPAN_KIND_RPC_SERVER;
-                    serverSpanTags[opentracing.Tags.PEER_HOST_IPV4] = ((127 << 24) | 1);
```

Comment:
This check will essentially become Utils.ip() == Utils.ip(). I'm not sure how the host ipv4 was hardcoded before...